libs/utils/utils.js

```diff
@@ -1270,4 +1270,32 @@ export function loadLana(options = {}) {
   window.addEventListener('unhandledrejection', lanaError);
 }
 
+const defaultAllowedOrigins = [
```

Comment:
The allowed origins are strictly related to federal content, so we either rename the constant or we move it within the method.